### ISSUE TITLE
Overdue Action Item Email Notification

### ISSUE THREAD

**Issue by wasimdf** [4minitz/4minitz#572]
Hi Team can we enable email notification for overdue action items for the respective member?

**Comment by emircankizilkaya** [comment]
did you solve this ? thank you

**Comment by wasimdf** [comment]
No i didn't find the solution for this.....have you fixed this?

**Comment by emircankizilkaya** [comment]
No. I guess that we have to add some codes to ActionItemsMailHandler.js  for overdue action item

**Comment by wasimdf** [comment]
oh...is it...have you tried that?

**Comment by emircankizilkaya** [comment]
i change some codes of this app for my company. And i have to add this property to app.

**Comment by emircankizilkaya** [comment]
> oh...is it...have you tried that?

no. maybe we will create new file for this. i dont know what will do exactly

**Comment by wasimdf** [comment]
oh...ok....if you find the solution please let me know..

**Comment by emircankizilkaya** [comment]
okey

**Comment by emircankizilkaya** [comment]
hey @wasimdf ı did this property for 4minitz. if you want to know how ,contact me.
